src/main/kotlin/com/ampnet/projectservice/service/pojo/OrganizationInvitationWithData.kt

```diff
@@ -7,11 +7,13 @@ import java.util.UUID
 data class OrganizationInvitationWithData(
     val organizationUuid: UUID,
     val organizationName: String,
-    val role: OrganizationRole
+    val role: OrganizationRole,
+    val organizationHeaderImage: String
```

Comment:
Why not `String?`? It is easier to handle `null` value than empty string

Reply:
I agree, I followed the convention on other responses. Check package com.ampnet.projectservice.controller.pojo.response.OrganizationResponse. It returns empty string rather than null.

Reply:
Ok, change headerImage to `String?` in all responses but leave `description` as empty string.